### PR TITLE
Display ERC 721/1155 transfers in a specific transaction

### PR DESCRIPTION
**Pull request by JeffreyMa597**
What problem does this PR solve?
------
Display ERC 721/1155 transfers in a specific transaction
Ref: https://github.com/Magickbase/godwoken_explorer/issues/969

we're going to add two tabs

1、ERC 721 Transfers
2、ERC 1155 Transfers  

Check List
------

### Test
unit test & e2e test
### Task
none

### CODE REVIEW

**Comment by FrederLu** [comment]
https://godwoken-explorer-gf7sb44zp-magickbase.vercel.app/zh-CN/tx/0x5fd6107f4427b0cd4b247f21534e8de8f96556d224341ad600a6f68802631708?tab=erc721
image
@JeffreyMa597  need to confirm.

**Review by qiweiii — APPROVED** [review]

**Comment by JeffreyMa597** [comment]
> https://godwoken-explorer-gf7sb44zp-magickbase.vercel.app/zh-CN/tx/0x5fd6107f4427b0cd4b247f21534e8de8f96556d224341ad600a6f68802631708?tab=erc721 image @JeffreyMa597 need to confirm.
> 
> Now is this pr only ERC721, ERC1155? I read the instructions on the requirements, but currently there is only ERC721 information on the page.

@FrederLu I'll make double-check whether we need to show the ERC1155 also. Thx.

File: components/CommonTransferlist/index.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+                      isShowValue
+                        ? `/token/${item.udt.id}`
+                        : `/nft-item/${item.token_contract_address_hash}/${item.token_id}`
+                    }
+                  >
+                    <a>
+                      {isShowLogo ? (
+                        <TokenLogo name={item.udt.name} logo={item.udt.icon} />
+                      ) : (
+                        item.udt.symbol.split('.')[0] ?? ''
```
Comment (Keith-CY) [inline components/CommonTransferlist/index.tsx:141]: Seems that NFT's id is not shown

**Comment by JeffreyMa597** [comment]
> > https://godwoken-explorer-gf7sb44zp-magickbase.vercel.app/zh-CN/tx/0x5fd6107f4427b0cd4b247f21534e8de8f96556d224341ad600a6f68802631708?tab=erc721 image @JeffreyMa597 need to confirm.
> > Now is this pr only ERC721, ERC1155? I read the instructions on the requirements, but currently there is only ERC721 information on the page.
> 
> @FrederLu I'll make double-check whether we need to show the ERC1155 also. Thx.

@FrederLu After confirmed, we will not implement ERC1155 in this ticket.

**Comment by JeffreyMa597** [comment]
@FrederLu @Naupio Please have a look at this issue. Our quest parameters are shown below. 
image

**Comment by Keith-CY** [comment]
https://godwoken-explorer-ui-git-feat-display-erc-721-724031-magickbase.vercel.app/tx/0xdffeaa791de34c1dc5642bd420007dc6234967a2c6d91bf95b7e08aa3524fb6a?tab=erc721Records
image
All records have log index `1` which is impossible, each record should have its own event log.

**Comment by Keith-CY** [comment]
Token ID is incorrect
image
The id is 0 but `5845` is displayed

**Comment by Keith-CY** [comment]
> [godwoken-explorer-ui-git-feat-display-erc-721-724031-magickbase.vercel.app/tx/0xdffeaa791de34c1dc5642bd420007dc6234967a2c6d91bf95b7e08aa3524fb6a?tab=erc721Records](https://godwoken-explorer-ui-git-feat-display-erc-721-724031-magickbase.vercel.app/tx/0xdffeaa791de34c1dc5642bd420007dc6234967a2c6d91bf95b7e08aa3524fb6a?tab=erc721Records) image All records have log index `1` which is impossible, each record should have its own event log.

These records are from various transactions so they have the same log index `1`
image

**Comment by JeffreyMa597** [comment]
@Naupio So, it seems we didn't get the perfect result if we use the field `to_address` to get the data of the erc721 transaction list. 
Could you please tell me the whole parameters that we need except `before` and `after`?

**Comment by JeffreyMa597** [comment]
> 

I have changed it.

**Comment by JeffreyMa597** [comment]
> Token ID is incorrect image The id is 0 but `5845` is displayed

Fixed.

**Comment by Naupio** [comment]
> @FrederLu @Naupio Please have a look at this issue. Our quest parameters are shown below. image

if combine_from_to is true, then from_address and to_address are combined into query condition like `address = from_address OR address = to_address`

by default, `combine_from_to` is true

if you want to searching the result like `address = from_address AND address = to_address` strategy, must adding `combine_from_to`  with `false` to input feilds

**Comment by Naupio** [comment]
> @Naupio So, it seems we didn't get the perfect result if we use the field `to_address` as the key parameter to get the data of the erc721 transaction list. Could you please tell me the whole parameters that we need except `before`, `after`, `limit`, and `sorter`? Thanks a lot.
> 
> image

```
    field :after, :string,
      description: "An opaque cursor representing the last row of the current page."

    field :before, :string,
      description: "An opaque cursor representing the first row of the current page."

    field :limit, :integer,
      description: "The maximum number of entries that can be contained in this page."

    field :total_count, :integer, description: "The total number of entries matching the query."
```

```
  input_object :paginate_input do
    @desc "Fetch the records before this cursor."
    field :before, :string
    @desc "Fetch the records after this cursor."
    field :after, :string

    @desc "Limits the number of records returned per page. Note that this number will be capped by maximum_limit=100. Defaults to 20."
    field :limit, :integer, default_value: 20
  end
```

sorter use for custom ordering strategy

**Comment by JeffreyMa597** [comment]
@Naupio  I have completed the request of ERC721 and ERC1155 based on the request method of ERC20list. Their request methods and parameters, in my opinion, seem to be the same.
If I made a mistake, please let me know. Thanks.

image

**Review by qiweiii — APPROVED** [review]

**Review by FrederLu — APPROVED** [review]

**Pull request merged**